docs/navbar.html

```diff
@@ -13,7 +13,7 @@
 
       <ul class="nav-menu">
         <a href="./"><li>Map</li></a>
-        <a href="./import/add_disaster.html"><li>Add Disaster</li></a>
+        <a href="import/manage_layers.html"><li>Add Disaster</li></a>
```

Comment:
Probably shouldn't be Add Disaster anymore right? And add link for manage_disaster?

Reply:
Changed to Manage Layers. Will leave up to @ruthtalbot if we should have an add disaster link here in a follow-up.

Reply:
Let's ask Ruth when she comes back but I lean towards wanting all the pages linked.